### PR TITLE
get rid of several @browser calls

### PR DESCRIPTION
**Pull request by WMDE-Fisch**

### CODE REVIEW

**Comment by WMDE-Fisch** [comment]
something went wrong, here is the old request for this ;-) :

https://github.com/wmde/WikidataBrowserTests/pull/64

**Comment by zeljkofilipin** [comment]
:+1:

File: tests/browser/features/support/modules/entity_module.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def wait_for_entity_to_load
 
   def wait_for_label(label)
     wait_until do
-      @browser.title.include?(label)
+      title.include?(label)
```
Comment (thiemowmde) [inline tests/browser/features/support/modules/entity_module.rb]: I suggest to keep this for readability. "title" alone is very generic and could mean anything.

**Comment by WMDE-Fisch** [comment]
plz review & merge @thiemowmde

**Pull request merged**